packages/react-composites/src/composites/CallWithChatComposite/components/DialpadModal.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { useState } from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { useMemo } from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { Dialpad, DialpadStyles, _DrawerMenu, _DrawerMenuItemProps, _DrawerSurface } from '@internal/react-components';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { IButtonStyles, IModalStyles, Modal, Stack, useTheme, Text, IconButton, PrimaryButton } from '@fluentui/react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { Call20Regular } from '@fluentui/react-icons';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { drawerContainerStyles } from '../styles/CallWithChatCompositeStyles';
+
+/** @private */
+export interface DialpadModalStrings {
+  dialpadModalAriaLabel: string;
+  DialpadModalTitle: string;
+  closeModalButtonAriaLabel: string;
+  callButtonLabel: string;
+}
+
+/** @private */
+export interface DialpadModalProps {
+  isMobile: boolean;
+  isModalOpen: boolean;
+  strings: DialpadModalStrings;
+  hideModal: () => void;
+}
+
+/** @private */
+export const DialpadModal = (props: DialpadModalProps): JSX.Element => {
+  /* @conditional-compile-remove(PeoplePaneDropdown) */
```

Comment:
This file needs no conditional compilation at all.

The live documentation for conditional compilation explains how to keep conditional compilation to a minimum: https://github.com/Azure/communication-ui-library/blob/main/packages/acs-ui-common/src/conditional-compilation-sample/index.tsx#L30

Since `DialpadModal` is a `@private` component, we should define it unconditionally. Only the `PeoplePaneContent` should have (minimal) conditional compilation necessary to avoid rendering the `DialpadModal` in stable builds.

Reply:
Updated, however still need to conditional compile part of this because the dialpad component is conditionally compiled